wiki/blocks/first-block.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Adding your first Block
+index: 1
+---
+# Adding your first Block
+
+Creating a Block is pretty similar to creating an Item, [read the corresponding article](../items/first-item) for more elaborate explanations.
+
+## Creating and registering the Block
+
+First we create the Block and store it in a field:
+
+```java
+public final Block EXAMPLE_BLOCK = new Block(new QuiltBlockSettings())
+```
+
+Then we register it in the `onInitialize()` function:
+
+```java
+Registry.register(Registries.BLOCK, new Identifier(mod.metadata().id(), "example_block"), EXAMPLE_BLOCK);
+```
+
+## Adding a BlockItem
+
+Having done that, we can place the block using the `setblock` command, but there isn't an item to place it. So we additionally register a `BlockItem` for the block and add it to the `BUILDING_BLOCKS` item group:
+
+```java
+Registry.register(Registries.ITEM, new Identifier(mod.metadata().id(), "example_block"), new BlockItem(EXAMPLE_BLOCK, new QuiltItemSettings()))
+
+ItemGroupEvents.modifyEntriesEvent(ItemGroups.BUILDING_BLOCKS).register(entries -> {
+	entries.addItem(EXAMPLE_BLOCK.asItem());
+});
+```
+
+For more information on what this does, see the [Creating your first Item](../items/first-item#registering-the-item) Article.
+
+## Adding a Model for the Block
+
+First, we need to create the blockstates file for your Block. This tells the game which `BlockState`s use which models. For an example on using a more complicated `BlockState` setup, see [Adding Redstone functionality to your Block](redstone-interaction). In this case, we only have one `BlockState` and can use this simple JSON:
+
+`assets/simple_block_mod/blockstates/example_block.json`:
+
+```json
+{
+	"variants": {
+		"": {
+			"model": "simple_block_mod:block/example_block"
+		}
+	}
+}
+```
+
+Replace `simple_block_mod` and `example_block` with your mod id and block name. Be sure to replace them in both the file/folder names and the JSON file.
```

Comment:
I'd toss this, we aren't telling the readers to change the names anywhere else, meaning it's possible someone sees this, renames the json nonsense, but leaves it registered as `example_block`

Reply:
Rather than get rid of this, I think we should specify it everywhere it's needed.
Failing to update these in all the necessary places seems like a very common 'gotcha' for new modders.

Reply:
^ I agree with this actually. please add it to the block registration!